features/environment.py

```diff
@@ -12,7 +13,6 @@
 from testsuite.config_reader import get_vendor_config, get_env_config
 from testsuite.oauth import authorize, factory
 
-
 CACHE_TTL = 60 * 60  # 1 hour
 CCDS_TAGS = {
```

Comment:
This is now a bit of a misnomer -- probably better to call these `FHIR_RESOURCE_TAGS`.

Reply:
Done!